### ISSUE TITLE
tlint problems validating check 'id's

### ISSUE THREAD

**Issue by angelabriel** [trento-project/tlint#5]
tlint 0.9.2
When the check id contains only digits, but no other character, tlint reports an error
  790926   - 790926 is not of type "string"
  path: /id

it is ignoring '0'
  7E0221   - 7e221 is not of type "string"
  path: /id

and really confusing
  822E47   - 8.22e49 is not of type "string"
  path: /id

**Comment by arbulu89** [comment]
@angelabriel Most probably we can fix this adding the check id in quotes. Like `"790926"`, etc
Could you have a look?
By the way, using these kind of ids breaks as well the frontend.
I already opened an internal ticket, but maybe simply using quotes will be a good solution

**Comment by angelabriel** [comment]
> @angelabriel Most probably we can fix this adding the check id in quotes. Like `"790926"`, etc Could you have a look? 

@arbulu89 yes, using quotes definitely fixes the issue. So I used quotes on all the checks I currently submitted to wanda. And I added the quotes too to our script, which creates check templates from the confluence checks tables. So with that we hopefully will not forget about this 'feature' :smile:

**Comment by fabriziosestito** [comment]
@angela @arbulu89 in this case I am afraid that tlint is right. An ambiguous string in yaml is always quoted, or it could be misinterpreted for other data,  so the failure is "excepted". 
See here for reference: https://symfony.com/doc/current/components/yaml/yaml_format.html

**Comment by angelabriel** [comment]
>  See here for reference: https://symfony.com/doc/current/components/yaml/yaml_format.html

Thanks for this really helpful link. Will take a closer look to get a little bit more familiar with the yaml format. Till now I slightly neglected this a bit.
